.rubocop.yml

```diff
@@ -3,11 +3,14 @@ require:
   - rubocop-performance
 
 AllCops:
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.4
   DisplayCopNames: true
   Exclude:
     - bin/**/*
 
+Metrics:
+  Enabled: false
```

Comment:
I don't like Metrics cops because they often introduce unnecessary refactoring, such as method splitting. So I disabled the department.